packages/testing/config/jest/api/jest.setup.js

```diff
@@ -78,9 +78,9 @@ const getQuoteStyle = async () => {
 }
 
 const getProjectDb = () => {
-  const { db } = require(`${apiSrcPath}/lib/db`)
+  const { db, testDb } = require(`${apiSrcPath}/lib/db`)
 
-  return db
+  return process.env.TEST_DATABASE_STRATEGY === 'rls' ? testDb : db
```

Comment:
See other comment, but I wonder if the scenario can be given a config to take in the Prisma Client(s) to use when creating db and running the tests instead of introducing another strategy.